### ISSUE TITLE
If there's no query string then request.args isn't a RequestParameters object

### ISSUE THREAD

**Issue by WilliamMayor** [channelcat/sanic#452]
So code like this:

```
fruits = request.args.getlist('fruit', ['apple', 'pear'])
```

Fails with this error:

```
2017-02-16 21:43:34,393: ERROR: Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/sanic/sanic.py", line 329, in handle_request
    response = await response
  File "/usr/src/app/tracker/api.py", line 23, in time
    fruits = request.args.getlist('fruit', ['apple', 'pear'])
AttributeError: 'dict' object has no attribute 'getlist'
```

If the URL doesn't contain a query string.

Although I've only just picked up this library tonight... so I might have missed something obvious.

**Comment by seemethere** [comment]
Just put up a PR for it, should be in for v `0.4.0`

**Comment by WilliamMayor** [comment]
Lovely, thank you!

**Comment by seemethere** [comment]
Closed with #454
